pom.xml

```diff
@@ -85,6 +86,23 @@
             <artifactId>rest-service-common-library</artifactId>
             <version>${rest-service-common-library.version}</version>
         </dependency>
+        <!--  remove these 3 log4j once spring boot upgrade is available after 23/12/2021 -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.15.0</version>
```

Comment:
will need to upgrade to 2.16.0 as apparently 2.15.0 has a security flaw

Reply:
Upgraded.